docs/experimental-features.md

```diff
@@ -4,7 +4,7 @@ Experimental features are those features that work, but haven't been thoroughly
 
 ## Rollback Configuration
 
-Starting in version 2.0.2, a featured called rollback configuraiton was introduced. The rollback configuration is exactly what it sounds like. It renders a rollback configuration in the event that a remediation causes a hiccup when being deployed. The rollback configuration does the inverse on a remediation. Instead of a remediation being renedered based upon the generated config, a rollback remediation is rendered from the generated config based upon the running configuration.
+Rollback configuration generation was introduced in version 2.0.2. The rollback configuration is exactly what it sounds like. It renders a rollback configuration in the event that a remediation causes a hiccup when being deployed. The rollback configuration does the inverse on a remediation. Instead of a remediation being rendered based upon the generated config, a rollback remediation is rendered from the generated config based upon the running configuration.
```

Comment:
May I suggest:

Rollback Configuration Generation was introduced in version 2.0.2. It renders the inverse of a remediation (i.e. the rollback configuration) in the event that a remediation causes a hiccup when being deployed. A rollback configuration is rendered from the generated configuration  based on the running configuration instead of the opposite like a remediation would be.